src/main/java/org/stellar/anchor/paymentsservice/Account.java

```diff
@@ -1,9 +1,12 @@
 package org.stellar.anchor.paymentsservice;
```

Comment:
Can we use org.stellar.anchor.paymentservice?

Reply:
Thanks for pointing that out! Renamed in https://github.com/stellar/java-stellar-anchor-sdk/pull/2/commits/4d89efa47b70e283b7b29830658ade547009e7ef